big-red-shuttle/StopsViewController.swift

```diff
@@ -60,8 +62,43 @@ class StopsViewController: UIViewController, UITableViewDelegate, UITableViewDat
         let locations = stops.map { $0.getCoordinate() }
         setLocations(locations: locations)
         viewIsSetup = true
+        
+        //observe gps data changes
+        system().gps().addObserver(observer: self)
+        Location.shared.fetchedUserLocationCompletionBlock = {
+            if !system().gps().registeredToLogLocation {
+                system().api().registerUserToLogLocation(key: "fedorko", success: { (json: JSON?) in
+                    system().gps().startLoggingShuttleLocation()
+                    system().gps().startFetchingShuttleLocation()
```

Comment:
Why does it have to be in 'every class.' Storing it off to a `gps` variable once is also much better than dispatching 2 method calls each time.